blacken_docs.py

```diff
@@ -22,7 +22,7 @@
 PY_LANGS = '(python|py|sage|python3|py3|numpy)'
 RST_RE = re.compile(
     rf'(?P<before>'
-    rf'^(?P<indent> *)\.\. (code|code-block|sourcecode|ipython):: {PY_LANGS}\n'
+    rf'^(?P<indent> *)\.\. (jupyter-execute::|(code|code-block|sourcecode|ipython):: {PY_LANGS})\n'
```

Comment:
looks like this line ends up being too long as well -- I might split this up by putting above:

```python
BLOCK_TYPES = '(code|code-block|sourcecode|ipython)'
```
and then swapping that part out for `{BLOCK_TYPES}` inside the regex

:+1:

Reply:
Done!